phylokit/dataset.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+import fsspec
+import xarray as xr
+
+
+def open_dataset(store, **kwargs):
+    """
+    Open dataset from zarr store
+
+    :param store: zarr store
+    :param kwargs: keyword arguments to pass to xarray.open_zarr
+    """
+    return xr.open_zarr(store, consolidated=True, concat_characters=False, **kwargs)
+
+
+def save_dataset(ds, store, storage_options=None, **kwargs):
+    """
+    Save dataset to zarr store
+
+    :param ds: xarray dataset
+    :param store: zarr store
+    :param kwargs: keyword arguments to pass to ds.to_zarr
+    """
+    if isinstance(store, str):
+        storage_options = storage_options or {}
+        store = fsspec.get_mapper(store, **storage_options)
+    elif isinstance(store, Path):
```

Comment:
We prefer the form ``pathlib.Path`` here, makes it clearer where things are coming from locally.